src/main/java/oshi/util/ExecutingCommand.java

```diff
@@ -35,6 +33,7 @@
 		} catch (IOException e) {
 			return null;
 		}
+		p.destroy();
```

Comment:
removed p.waitFor(); so it works with top command. 
p.destroy(); just to be safe, that process is dead.